python/ml4ir/base/config/parse_args.py

```diff
@@ -281,6 +295,13 @@ def define_args(self):
             help="String specifying the file handler to be used. Should be one of FileHandler keys in ml4ir/base/config/keys.py",
         )
 
+        self.add_argument(
+            "--track_experiment",
+            type=bool,
+            default=False,
+            help="Whether to track model performance and hyperparameters.",
```

Comment:
Can we say where the user can access to the stats?

Reply:
I left this out on purpose. Currently we use weights and biases, but we might move out of this and just use a simple JSON file.

Reply:
So in the help we need to say what we currently do, and update it when we change it :)

Reply:
@lastmansleeping +1 Very minor but would be helpful to track what we currently do and update when we change it.